packages/pluggableWidgets/bottom-sheet-native/src/BottomSheet.editorConfig.ts

```diff
@@ -9,85 +9,64 @@ import {
 
 import { BottomSheetPreviewProps } from "../typings/BottomSheetProps";
 
-export function getPreview(values: BottomSheetPreviewProps, isDarkMode: boolean): StructurePreviewProps {
-    const contentFontColor = isDarkMode ? "#DEDEDE" : "#6B707B";
-    return {
-        type: "Container",
-        borders: true,
-        children: [
-            {
-                type: "Container",
-                backgroundColor: isDarkMode ? "#454545" : "#F5F5F5",
-                children: [
-                    {
-                        type: "Container",
-                        padding: 4,
-                        children: [
-                            {
-                                type: "Text",
-                                fontColor: contentFontColor,
-                                content: "Bottom sheet"
-                            }
-                        ]
-                    }
-                ]
-            },
-            ...((values.type === "modal"
-                ? values.modalRendering === "custom"
-                    ? [
-                          {
-                              type: "DropZone",
-                              property: values.largeContent,
-                              placeholder: "Content"
-                          }
-                      ]
-                    : values.itemsBasic.map((value, index) => ({
-                          type: "RowLayout",
-                          columnSize: "grow",
-                          padding: 12,
-                          borders: true,
-                          children: [
-                              {
-                                  type: "Container",
-                                  grow: 1
-                              },
-                              {
-                                  type: "Text",
-                                  fontColor: contentFontColor,
-                                  content: value.caption || `[Item ${index + 1}]`
-                              },
-                              {
-                                  type: "Container",
-                                  grow: 1
-                              }
-                          ]
-                      }))
-                : [
-                      {
-                          type: "DropZone",
-                          property: values.smallContent,
-                          placeholder: "Always visible"
-                      },
+export const getPreview = (values: BottomSheetPreviewProps, isDarkMode: boolean): StructurePreviewProps => {
+    const content: StructurePreviewProps[] = (
+        values.type === "modal"
+            ? values.modalRendering === "custom"
```

Comment:
It is better to avoid nested ternaries as they make it harder to follow the code